client/landing/jetpack-cloud/components/sidebar/style.scss

```diff
@@ -1,42 +1,45 @@
 // Menu links
-.sidebar__menu-link {
-	padding-top: 14px;
-	padding-bottom: 14px;
-	font-weight: 600;
-
-	.selected &,
-	.selected &:hover {
-		background-color: rgba( var( --color-sidebar-menu-selected-background-rgb ), 0.12 );
-	}
-}
+.is-jetpack-cloud-sidebar {
```

Comment:
According to what I proposed above, this line would have to be `.sidebar--jetpack-cloud`.

Reply:
👍